src/api/resources/enumerations.py

```diff
@@ -0,0 +1,33 @@
+"""
+Stores enumerations used by API resources.
+"""
+
+from enum import Enum
+
+# Enumeration for the train status of a model
+class RunwaySurface(Enum):
```

Comment:
I'd be inclined to move this class to `runways.py` as they're closely linked and rely on each other